docs/templating_vars.yaml

```diff
@@ -3,9 +3,9 @@ helm:
   repo_name: stackable-stable
   repo_url: https://repo.stackable.tech/repository/helm-stable/
 versions:
-  commons: 24.11.1-rc2
-  secret: 24.11.1-rc2
-  listener: 24.11.1-rc2
-  airflow: 24.11.1-rc2
+  commons: 24.11.1
+  secret: 24.11.1
+  listener: 24.11.1
+  airflow: 24.11.1
```

Comment:
Note to self, we should check the regexes across the scripts to ensure they account for pre-release info. Maybe that's already been done.

Reply:
I don't think the getting-started tests will work prior to release-tagging, as the images will only exist in their `-prXXX` state.

Reply:
We would have to manually edit that file to test that, much like we have done for the integration tests (see the release.yaml changes etc.)

Reply:
Will mark this as resolved, as I do think the regexes cover the pre-release info.